### ISSUE TITLE
Advanced Search

### ISSUE THREAD

**Issue by davidamichelson** [srophe/britishLibrary#352]
We should add a shelfmark field. What do you think?

**Comment by wlpotter** [comment]
Yes, this would be useful; I can add the xpath for this when I get a chance today or tomorrow

**Comment by wlpotter** [comment]
This may need more discussion. If the user knows the shelf mark wouldn't they be able to just go to that record and do their searching? There may be case for it when doing wild card searching (e.g., 14,4*) but I'm not sure of use cases for that 

Maybe this is more like an option in the basic search to either search everything or search just shelf mark? (compare the basic SMDL search: https://sinaimanuscripts.library.ucla.edu/catalog?utf8=%E2%9C%93&q=&search_field=all_fields)

In other words, we could have a menu in the basic search where users select "all fields" (the default), "shelfmark", etc.

**Comment by wlpotter** [comment]
#376 is for adding the advanced search. I'll make a separate issue for discussing whether we should have the option to select fields to search in from the basic search
